modules/reference/pages/properties/object-storage-properties.adoc

```diff
@@ -317,7 +317,17 @@ Divide the object storage cache across the specified number of buckets. This onl
 
 === cloud_storage_cache_size
 
-Maximum size of object storage cache. If both this property and <<cloud_storage_cache_size_percent,cloud_storage_cache_size_percent>> are set, Redpanda uses the minimum of the two.
+Maximum size of the object storage cache, in bytes.
+
+This property works together with <<cloud_storage_cache_size_percent,`cloud_storage_cache_size_percent`>> to define cache behavior:
+
+- When both properties are set, Redpanda uses the smaller calculated value of the two, in bytes.
+
+- If one of these properties is set to `0`, Redpanda uses the non-zero value.
+
+- These properties cannot both be `0`.
```

Comment:
In addition, `cloud_storage_cache_size` cannot be 0 while `cloud_storage_cache_size_percent` is `null` (empty optional)

Reply:
```suggestion
- These properties cannot both be `0`.

- `cloud_storage_cache_size` cannot be `0` while `cloud_storage_cache_size_percent` is `null`.
```